src/components/content-tab-assistant.tsx

```diff
@@ -73,31 +84,81 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 		}
 	}, [ messages ] );
 
+	useEffect( () => {
+		if ( isAuthenticated && inputRef.current ) {
+			inputRef.current.focus();
+		}
+	}, [ isAuthenticated ] );
+
+	const disabled = isOffline || ! isAuthenticated;
 	return (
 		<div className="h-full flex flex-col bg-gray-50">
-			<div data-testid="assistant-chat" className="flex-1 overflow-y-auto p-8">
-				<div className="text-gray-500 mb-4">
-					Welcome to the Studio assistant. I can help manage your site, debug issues, and navigate
-					your way around the WordPress ecosystem.
-				</div>
-				<div>
-					{ messages.map( ( message, index ) => (
-						<Message key={ index } isUser={ message.role === 'user' }>
-							{ message.content }
-						</Message>
-					) ) }
-					<div ref={ endOfMessagesRef } />
-				</div>
+			<div
+				data-testid="assistant-chat"
+				className={ cx(
+					'flex-1 overflow-y-auto px-8 py-4',
+					! isAuthenticated && 'flex items-end'
+				) }
+			>
+				{ isAuthenticated ? (
+					<>
+						<div className="text-gray-500 mb-4">
+							Welcome to the Studio assistant. I can help manage your site, debug issues, and
+							navigate your way around the WordPress ecosystem.
+						</div>
+						<div>
+							{ messages.map( ( message, index ) => (
+								<Message key={ index } isUser={ message.role === 'user' }>
+									{ message.content }
+								</Message>
+							) ) }
+							<div ref={ endOfMessagesRef } />
+						</div>
+					</>
+				) : (
+					<Message className="w-full" isUser={ false }>
+						<p className="mb-1.5 a8c-label-semibold">{ __( 'Hold up!' ) }</p>
+						<p>
+							{ __( 'You need to log in to your WordPress.com account to use the assistant.' ) }
+						</p>
+						<p className="mb-1.5">
+							{ createInterpolateElement(
+								__( "If you don't have an account yet, <a>create one for free</a>" ),
```

Comment:
It feels odd to punctuate one of two sentences. WDYT about adding a period to the second sentence? 
```suggestion
								__( "If you don't have an account yet, <a>create one for free</a>." ),
```